src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,19 +1,72 @@
+import React from 'react';
+import { CompletedFilter } from '../../types/CompletedFilter';
+import { getNameInCapitalizedCase } from '../../utils/getNameInCapitalizedCase';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  completedFilter: CompletedFilter;
+  setCompletedFilter: React.Dispatch<React.SetStateAction<CompletedFilter>>;
+  searchQuary: string;
+  setSearchQuary: React.Dispatch<React.SetStateAction<string>>;
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<Props> = ({
+  completedFilter,
+  setCompletedFilter,
+  searchQuary,
+  setSearchQuary,
+}) => {
+  const handleSelectComplitedFilter = (
```

Comment:
The method name `handleSelectComplitedFilter` also contains a typo. It should be `handleSelectCompletedFilter`.